extensions/positron-python/src/client/interpreter/configuration/interpreterSelector/commands/setInterpreter.ts

```diff
@@ -3,6 +3,9 @@
 
 'use strict';
 
+// --- Start Positron ---
+import * as fs from 'fs-extra';

```

Comment:
Nit: something I've seen us do with imports in upstream files is to include a single `// --- Start Positron ---` block _after_ their imports and add the relevant lines to ignore specific eslint errors.